### ISSUE TITLE
🐛 Development-setup installation fails while getting pnpm dependencies

### ISSUE THREAD

**Issue by ou-li** [mawoka-myblock/ClassQuiz#307]
### Which component is affected?

pnpm

### Did the issue occur at ClassQuiz.de, or on a self-hosted instance?

On a self-hosted instance

### How can the issue be reproduced?

Setup an OVH cloud instance (Model D2-2 RAM 2 GB Processor 1 vCores Storage 25 GiB ) tested both Fedora 38 and Ubuntu 23.04)

Install git (2.43.0) , pip (22.3.1) via system repository.
Install pipenv (2023.11.15) via pip install
Install caddy ()v2.7.5 via caddy repository https://caddyserver.com/docs/install
Install Docker Engine (24.0.7) via Docker repository, i.e. https://docs.docker.com/engine/install/fedora/

Follow the Setup instructions:  https://classquiz.de/docs/develop

### Describe the bug (with screenshots if possible)

`pnpm i`
fails:  ELIFECYCLE  Command failed.

![image](https://github.com/mawoka-myblock/ClassQuiz/assets/7612981/bb3470ce-26e9-4ec1-bce2-194254962fc0)

Where did I take a wrong turn? Do I need a physical server?

Thx for this beautiful project!

### Device

Desktop

### Operating System

Linux, Fedora 38, Ubuntu 23.04

### Browser

not related to browser use

**Comment by mawoka-myblock** [comment]
Will check that out. @Dave879 did you encounter this issue?

**Comment by Dave879** [comment]
No, everything got installed without any issues (Ubuntu 22.04.3 LTS) using pnpm 8.10.5

**Comment by ou-li** [comment]
Thx!
I'm on pnpm v8.11.0
installed via the POSIX script here: https://pnpm.io/fr/installation

I will check if downgrading pnpm makes any difference.

**Comment by ou-li** [comment]
Trying to set up everything *extra carefully* I'm still stuck with the same error now using python 3.**11** and pnpm v8.10.5.

Here's what looked suspicious to me:
-Installing everything as root is ok? (pip complains about it)
-Docker **Engine** is fine?
-Is this step necessary:
` To activate this project's virtualenv, run pipenv shell`
@Dave879 :
-What hardware are you on?

**Comment by mawoka-myblock** [comment]
> Trying to set up everything _extra carefully_ I'm still stuck with the same error now using python 3.**11** and pnpm v8.10.5.
> 
> Here's what looked suspicious to me: -Installing everything as root is ok? (pip complains about it) -Docker **Engine** is fine? -Is this step necessary: ` To activate this project's virtualenv, run pipenv shell` @Dave879 : -What hardware are you on?

Installing ad root could be the cause of the problem

**Comment by Dave879** [comment]
Yep, @mawoka-myblock is right, I can reproduce the issue by running the installation as root in a VM (didn't try on my local machine), but when running it as a normal user everything installs fine (just talking about frontend/sveltekit project, I didn't try anything else). (pnpm v8.11.0)

**Comment by mawoka-myblock** [comment]
Then, I'd consider this as closed, since working as root isn't expected nor recommendend nor normal.

**Comment by ou-li** [comment]
Thanks a million 🤩 @Dave879 and @mawoka-myblock for looking into this. 
I have switched to a VM on my own machine, **avoiding root** and am still running into the same issue. 🤯

@Dave879 : May I ask where you get your python3.**11** from on ubuntu 22.04.3?
The version from ppa:deadsnakes/ppa seems to cause a little hickup in the package management.
(python3.11 is expected according to Pipfile in the main folder)

This must be some NOOB issue in my configuration.
